src/Publisher.ts

```diff
@@ -309,9 +309,9 @@ export default class Publisher {
         return publishedFrontMatter;
     }
 
-    addHomePageTag(baseFrontMatter: any, newFrontMatter: any) {
+    addPageTag(baseFrontMatter: any, newFrontMatter: any) {
         const publishedFrontMatter = { ...newFrontMatter };
-        if (baseFrontMatter && baseFrontMatter["dg-home"]) {
+        if (baseFrontMatter) {
```

Comment:
This implementation will cause all notes to get the "gardenEntry" tag. 
The first version of the plugin used the tag "gardenEntry" to know which note was to be used as the front page. This is still what the template is using to determine the frontpage, in order to keep backwards compatibility. The current version of the plugin uses dg-home: true, but that gets converted into a gardenEntry tag. 

So we still need the check for baseFrontMatter["dg-home"]. Additionally I think it would be best to filter the gardenEntry tag away from template loop, as I would guess 99,99% of people don't want that to show up on their front page. 

So summarized: Add gardenEntry tag if dg-home is set to true, then add all other tags specified by the user.

Reply:
Cool, fixing this right away.

Reply:
Fixed this.